editor/src/editor/layout/assets-browser/items/item.tsx

```diff
@@ -352,7 +354,11 @@ export class AssetsBrowserItem extends Component<IAssetsBrowserItemProps, IAsset
 
 				<ContextMenuSeparator />
 
-				{items}
+				{items.map((item, index) => (
+                    <React.Fragment key={`context-menu-item-${index}`}>
```

Comment:
Please `import { Fragment } from "react"` so we are homogeneous compared to other source files in the project